react-common/components/controls/Feedback/Feedback.tsx

```diff
@@ -6,48 +6,55 @@ import { Modal } from "../Modal";
 
 // both components require onClose because the feedback modal should close when the user clicks the "finish" button
 // this would not happen if the EventListener did not have a callback to close the modal
-interface IFeedbackModalProps {
-  feedbackConfig: ocv.IFeedbackConfig;
+interface IFeedbackProps  {
+  kind: ocv.FeedbackKind;
   frameId: string;
-  title: string;
-  onClose: () => void;
+  onClose?: () => void;
 }
 
 // right now, there are two kinds of feedback that I think could be valuable for our targets
 // generic and rating feedback, but we will likely want to expand this
-interface IFeedbackProps {
+interface IFeedbackModalProps {
   kind: ocv.FeedbackKind;
   onClose: () => void;
 }
 
 // Wrapper component of the feedback modal so kind can determine what feedback actually shows in the modal
-export const Feedback = (props: IFeedbackProps) => {
+export const FeedbackModal = (props: IFeedbackModalProps) => {
   const { kind, onClose } = props;
+  const title = kind === "generic" ? lf("Leave Feedback") : lf("Rate this activity");
```

Comment:
nit: I would swap this to check if kind === "rating" so the fallback is generic (i.e. "Leave Feedback") in the event of an unrecognized/new kind being introduced.